www/common/drive-ui.js

```diff
@@ -1250,7 +1250,7 @@ define([
             var secret = Hash.getSecrets(parsed.type, parsed.hash, data.password);
             var opts = {};
             if (isRo) { opts.view = true; }
-            var hash = Hash.getHiddenHashFromKeys(parsed.type, secret, opts);
+            var hash = Hash.getHiddenHashFromKeys(parsed.type, secret, opts, parsed.hashData.auditorKey);
```

Comment:
See previous comment, we shouldn't add keys to the "hidden hashes" (a.k.a. safe links).